### PR TITLE
Edits to prepare for the Workshop at the 2019 Seattle AAS Meeting

### PR DESCRIPTION
**Pull request by JuanCab**
I have updated the README.md file to reflect the particulars of the workshop (NOTE: I have NO CLUE as to the schedule during the day of the workshop).  

I also fixed some bugs in the Advanced Tables notebook, which was apparently using deprecated versions of make_random_gaussians_table and make_random_gaussians_image from photutils.

### CODE REVIEW

File: 06-Tables/Advanced_Tables.ipynb [hunk 1]
```diff
@@ -154,18 +150,22 @@
    },
    "outputs": [],
    "source": [
-    "sources = photutils.datasets.make_random_gaussians(\n",
-    "    flux_range=(1000, 10000), n_sources=5, \n",
-    "    xmean_range=(10, 190), ymean_range=(10, 190),\n",
-    "    xstddev_range=(2, 5), ystddev_range=(2,5),\n",
-    "    random_state=1)"
+    "from collections import OrderedDict\n",
```
Comment (eblur) [inline 06-Tables/Advanced_Tables.ipynb]: Does this need to be an `OrderedDict` object?

Using `dict` will do almost the same thing:
```
param_ranges = dict([('flux', [1000, 10000]), ('x_mean', [10, 190]), ... ])
```

File: 06-Tables/Advanced_Tables.ipynb [hunk 2]
```diff
@@ -545,8 +527,8 @@
     "    sources['x0'] = np.round(sources['x_mean']).astype(int)\n",
     "    sources['y0'] = np.round(sources['y_mean']).astype(int)\n",
     "    \n",
-    "    # Generate list of cutout images around each source\n",
-    "    img = photutils.datasets.make_gaussian_sources(image_shape=(200, 200), source_table=out)\n",
+    "    # Generate list of cutout images around each source (syntax fixed)\n",
```
Comment (eblur) [inline 06-Tables/Advanced_Tables.ipynb]: Remove _(syntax fixed)_ comment

Comment (JuanCab) [reply 06-Tables/Advanced_Tables.ipynb]: Sorry, I thought I caught all of those comments about syntax, this is not an issue to edit.

File: 06-Tables/Advanced_Tables.ipynb (hunk 1 above)
Comment (JuanCab) [reply 06-Tables/Advanced_Tables.ipynb]: Sorry about that, I was just basing this off a code example, I confirmed it works with dict(), so better to use that and avoid the unnecessary import of `OrderedDict`.

**Pull request merged**